dummydns.py

```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env python3
 
-import socket, struct, sys
+import socket, struct, sys, signal
 
-default_address_dummydns = ["", 53]
+default_address_dummydns = ["127.0.0.1", 53]
```

Comment:
I'd rather stick to empty, which accepts any address, and may be used to host this script on the internet.

Reply:
Windows needs this to understand that you're opening at 127.0.0.1. "localhost" doesn't work either.

Reply:
Interesting.